### PR TITLE
Implement blog card 

### PR DESCRIPTION
**Pull request by jebessaD**
blogcard

### CODE REVIEW

File: Web/starter_project_vanguard/components/blog/BlogCard.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            {blog.date}
+          </p>
+        </div>
+
+        <div className="flex flex-row space-x-2 my-3">
+          {blog.skills.map((skill, index) => {
+            return (
+              <div
+                key={index}
+                className="rounded-full bg-neutral-100 uppercase text-[0.6rem] font-semibold text-secondary-text py-1 px-3"
```
Comment (Segnicho) [inline Web/starter_project_vanguard/components/blog/BlogCard.tsx:55]: for the font size let's use the already existing tailwind classes

**Review by Segnicho — COMMENTED** [review]
Overall this looks nice to me I have left you some nits:)

File: Web/starter_project_vanguard/components/blog/BlogCard.tsx (hunk 1 above)
Comment (jebessaD) [reply Web/starter_project_vanguard/components/blog/BlogCard.tsx:55]: The smallest text size in tailwind is 'text-xs', which is 0.75 rem, but the font-size mentioned on the UI is less than text-xs. That is why I used it implicitly.

File: Web/starter_project_vanguard/tsconfig.json [hunk 2]
```diff
@@ -18,6 +18,11 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "components/blog/fetch copy.jsx"
```
Comment (ruthgdb) [inline Web/starter_project_vanguard/tsconfig.json:25]: [nit] if your not going to use this, you should remove it.

File: Web/starter_project_vanguard/types/blogs.ts
Comment (ruthgdb) [inline Web/starter_project_vanguard/types/blogs.ts:1]: Move this file to /blogs folder

**Review by ruthgdb — CHANGES_REQUESTED** [review]

**Review by ruthgdb — APPROVED** [review]
LGTM. Well done.

**Pull request merged**